packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -37,6 +37,7 @@ import { toolsMap } from './tool'
 import { Highlighter } from './highlighter'
 import { setFunctionalGroupsTooltip } from './utils/functionalGroupsTooltip'
 import { contextMenuInfo } from '../ui/views/components/ContextMenu/contextMenu.types'
+import { FloatingToolsPayload } from '../ui/state/floatingTools'
```

Comment:
I think it'd be better to put `FloatingToolsPayload` in `ui/state`, because `FloatingToolsPayload` is defined along with function `updateFloatingTools` which exists in `ui/state`. I'm curious what your understanding of producer and consumer is, because I'm confused about them